scripts/ansible/ansible.cfg

```diff
@@ -0,0 +1,9 @@
+[defaults]
+host_key_checking=False
```

Comment:
Why do we turn off host key checking?

Reply:
Because the VM will not have the ssh keys for the remote host in the container...